pkg/serializer/serializer.go

```diff
@@ -244,7 +244,7 @@ func (s *Serializer) SendSketch(sketches marshaler.Marshaler) error {
 		return nil
 	}
 
-	compress := false // TODO: enable compression once the backend supports it on this endpoint
+	compress := true
```

Comment:
We could remove the variable and directly use `true` in the function call (like in the rest of the file).

Also should we add an option for this ? In case there is a issue in the backend.
That would allow client to revert back to the old behavior (we don't need to document that option in the datadog.yaml, if we don't want users to disable it by mistake).

Reply:
For me, having the variable named is easier to follow than an unlabeled `true` or `false`. But I certainly defer to the agent folks here; if we do change it I'll change the handful of other places it's this way (`SendEvents` and `SendServiceChecks` as well).

I don't think we need an option to disable this. Support for compressed payloads is both working well and is standard for other payload types; I don't imagine wanting to turn it off.

Reply:
OK 👍